include/ichain.h

```diff
@@ -55,4 +58,40 @@ typedef enum {
 #define ICHAIN_CONTINUE 1
 #define ICHAIN_STOP     0
 
+void Lib_MemSet(u8* dest, size_t len, u8 val);
```

Comment:
The function prototypes except maybe `Actor_ProcessInitChain` should be in `z_lib.h`?

Reply:
fwiw I think putting ichain stuff in a z_lib header instead of a header for its own system is a regression, but it can be addressed later